### PR TITLE
add php72 support

### PR DESCRIPTION
**Pull request by nexbrad**
@clwells adds remi php72 support, idempotent test passed.

### CODE REVIEW

File: playbooks/setup.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@
     - { role: nexcess.php, php_prefix: "php56" }
     - { role: nexcess.php, php_prefix: "php70" }
     - { role: nexcess.php, php_prefix: "php71" }
+    - { role: nexcess.php, php_prefix: "php72" }
```
Comment (clwells) [inline playbooks/setup.yml]: We 100% need this but installing 4 PHPs by default seems like much.  Each one adds 8 - 10 minutes of install time.  Can we get a consensus on what PHP version are actually needed by default for cloudhosts @rfxn ?

**Comment by clwells** [comment]
@nexbrad - need a decision here.

File: playbooks/setup.yml (hunk 1 above)
Comment (nexbrad) [reply playbooks/setup.yml]: I feel we have to have it there. Magento 2 has plans to support PHP 7.2 in an upcoming release and   wordpress supports it now. Not having 7.2 available will just cause tickets over the coming year like 7.1 does now on our el6 plans.

**Comment by rfxn** [comment]
@clwells @nexbrad It has been about the norm for quite a long time that you run about 3-4 PHP versions on a shared platform stack. It's just the realities of PHP's release cycles and customers/applications not updating fast enough. If we were to drop anything in a purely cut complexity and build time, it would likely be PHP70. Older applications updating to support 7.x are skipping right over 7.0 and anything already supporting 7.x has pretty much moved onto 7.1 and now 7.2.

**Comment by rfxn** [comment]
@clwells we will have a decision on this between @nexbrad and myself by end of business today.

**Comment by rfxn** [comment]
@nexbrad and I discussed a decision on a release policy for new PHP versions:
major.minor.point
point 5< not fit for production
point >=5 through 8 = stable for as-needed deployments
>8 = release; generally available on all systems, consistently.

As such, we are good to close/say 7.2 will not be at release level for metaworx launch.

**Pull request closed without merging**